src/common/containers/landing.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import { Anchor } from '../components/button';
+
+import styles from './container.css';
+
+class Landing extends Component {
+  componentWillMount() {
+    if (this.props.auth.authenticated) {
```

Comment:
I don't understand the reason for having `/` redirect, or switch to, the dashboard. I don't think there is anything that qualifies build.snapcraft.io as a SPA, merely that it is a site that happens to be built using React. URLs are the conventional state of the location within an app, and this seemed like an obvious example for separating pages by route.

Take for instance the "select repos" dashboard. If this is also located on `/`, then a user can't use the browser's history buttons to return to the "list repos" dashboard. That's breaking the web.

If the redirect to the `/repos` route stands, I would like to suggest this redirect is done using an HTTP redirect in an Express handler instead. This follows convention and is where I would expect to look to find a landing page redirect.